auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py

```diff
@@ -1010,7 +1010,7 @@ def checkpoint_load(self, ckpt: str, model: torch.nn.Module, **kwargs):
             if self.global_rank == 0:
                 warnings.warn("Invalid checkpoint file: " + str(ckpt))
         else:
-            checkpoint = torch.load(ckpt, map_location="cpu")
+            checkpoint = torch.load(ckpt, map_location="cpu", weights_only=False)
```

Comment:
Same here. Thanks.

Reply:
Sure, added in this commit: https://github.com/Project-MONAI/research-contributions/pull/412/commits/50025344dc9d32818d687596259c9c8c37b9babe